### PR TITLE
pages with no direction fixed and payment is added

### PR DESCRIPTION
**Pull request by farahh001**

### CODE REVIEW

File: techeria_controller/templates/accessorie.html [hunk 1]
```diff
@@ -192,112 +192,141 @@ <h5 class="card-title">{{ p.name }}</h5>
...
-                                    <li><a href="#">Gadets</a></li>
-                                </ul>
-                            </div>
-                        </div>
+                      <div class="col-md-3 col-sm-6">
+                          <div class="footer-menu">
+                              <h2 class="footer-wid-title" style= "color: white">Categories</h2>
+                              <ul>
+                                  <li><a href="#">Mobile Phone</a></li>
+                                  <li><a href="#">Home accesseries</a></li>
```
Comment (lianahasan) [inline techeria_controller/templates/accessorie.html:284]: Check the spelling of accessories for the UI

File: techeria_controller/templates/contact.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
   <meta name="description" content="" />
```
Comment (lianahasan) [inline techeria_controller/templates/contact.html:13]: This contact page shouldn't have an image of a watch. I will add the change to make sure the contact page only has an image related to technology

File: techeria_controller/techeria_app/models.py [hunk 3]
```diff
@@ -182,62 +182,44 @@ class ShippingAddress(models.Model):
...
+
+class PlaceOrder(models.Model):
+    STATUS = (
+        ('New', 'New'),
+        ('Accepted', 'Accepted'),
+        ('Completed', 'Completed'),
+        ('Cancelled', 'Cancelled'),
+    )
+
+    user = models.ForeignKey(BuyerModel, on_delete=models.SET_NULL, null=True)
```
Comment (kabirulislam95) [inline techeria_controller/techeria_app/models.py:199]: Good as you created the foreign key that are taking from the buyer table

File: techeria_controller/techeria_app/models.py [hunk 4]
```diff
@@ -182,62 +182,44 @@ class ShippingAddress(models.Model):
...
-#     ordered = models.BooleanField(default=False)
-#     created_at = models.DateTimeField(auto_now_add=True)
-#     updated_at = models.DateTimeField(auto_now=True)
-
-#     def __str__(self):
-#         return self.product.product_name
 
+
+
+class PlaceOrder(models.Model):
```
Comment (lianahasan) [inline techeria_controller/techeria_app/models.py:191]: This should add an order to the orders database table, but it doesn't happen so we should take a closer look at this and make sure it works.

File: techeria_controller/techeria_app/views.py [hunk 5]
```diff
@@ -403,66 +403,43 @@ def get(self, request, uidb64, token):
...
+            data = PlaceOrder()
+            data.user = current_user
+            data.first_name = form.cleaned_data['first_name']
+            data.last_name = form.cleaned_data['last_name']
+            data.phone = form.cleaned_data['phone']
+            data.email = form.cleaned_data['email']
+            data.address_line_1 = form.cleaned_data['address_line_1']
+            data.address_line_2 = form.cleaned_data['address_line_2']
+            data.country = form.cleaned_data['country']
+            data.state = form.cleaned_data['state']
```
Comment (kabirulislam95) [inline techeria_controller/techeria_app/views.py:422]: good that things

File: techeria_controller/templates/checkout.html [hunk 6]
```diff
@@ -203,53 +203,58 @@ <h4>Information for One Product (from database):</h4>
...
                         <!--<button id="make-payment">Make payment</button>-->
                         <div id="paypal-button-container"></div>
+
+
+                        <!-- Include the PayPal JavaScript SDK -->
+                        <script src="https://www.paypal.com/sdk/js?client-id=test&currency=USD"></script>
+
+                        <script>
+                        // Render the PayPal button into #paypal-button-container
+                        paypal.Buttons({
```
Comment (kabirulislam95) [inline techeria_controller/templates/checkout.html:224]: Thanks for using paypal staff for save

File: techeria_controller/templates/contact.html [hunk 7]
```diff
@@ -103,66 +103,93 @@ <h2 class="custom_heading text-center">
...
+            <div class="row">
+                <div class="col-md-3 col-sm-6">
+                    <div class="footer-about-us">
+                        <h2><span style="color: white">Techeria</span></h2>
+                        <h2><span style="color: white">Our Values</span></h2>
+                        <p>We are an e-commerce application, focused on making sure all technology products can be purchased in one space with the best deals so you can save your precious time without having to look through multiple websites.</p>
+                        <div class="footer-social">
+                            <a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+                            <a href="#" target="_blank"><i class="fa fa-twitter"></i></a>
+                            <a href="#" target="_blank"><i class="fa fa-youtube"></i></a>
```
Comment (kabirulislam95) [inline techeria_controller/templates/contact.html:172]: home page for every pages

**Comment by sanjidanisha** [comment]
All the reviews are good

**Pull request merged**